src/ecrc-frontend/src/components/base/sideCard/SideCard.css

```diff
@@ -74,7 +80,12 @@
   color: #fff;
   margin-bottom: 20px;
   padding: 15px;
-  max-width: 400px;
+  /* max-width: 400px; */
+}
+
+.bluegrey-container {
+  margin-bottom: 20px;
+  /* max-width: 400px; */
```

Comment:
same here?

Reply:
Yeah. I guess I forgot all these were here while testing if I could take them out

Reply:
that's why reviewing code is useful! :)